Add ability to run on worker nodes (#4)
giantswarm/k8s-initiator-app: README.md

# k8s-initiator-app

> An application that runs as daemonset only in the master nodes (by default) to maintain the state of some Kubernetes resources through cluster upgrades or master replacements.

Reviewer: ```suggestion
An application that runs as daemonset (masters and/or workers) to maintain the state of some Kubernetes resources through cluster upgrades or master replacements.
```

Reviewer: IMO it should run in all nodes by default, and you can customize only workers or master via values.yaml, WDYT?
